packages/jellyfish-api-core/__tests__/category/masternode/unsetGov.test.ts

```diff
@@ -0,0 +1,73 @@
+import { RpcApiError } from '@defichain/jellyfish-api-core'
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { createToken } from '@defichain/testing'
+import { ContainerAdapterClient } from '../../container_adapter_client'
+
+describe('Masternode', () => {
+  const container = new MasterNodeRegTestContainer()
+  const client = new ContainerAdapterClient(container)
+
+  beforeAll(async () => {
+    await container.start()
+    await container.waitForReady()
+    await container.waitForWalletCoinbaseMaturity()
+    await container.generate(20)
+
+    await createToken(container, 'CAT')
+  })
+
+  afterAll(async () => {
+    await container.stop()
+  })
+
+  it('should throw error if not enabled', async () => {
```

Comment:
Maybe can add another test case to cover the case for when a non-set governance variable / attributes is unset.